src/app.scss

```diff
@@ -227,4 +227,42 @@ a {
 .slide-in-top-out {
   animation-direction: reverse;
 }
+
+.custom-overlay{
+  background-image: linear-gradient(to bottom, rgba(237, 243, 247, 0.8), rgba(237, 243, 247, 0.8)) !important;
+}
+
+.networks-wrap{
+  max-width: 1040px;
+  width: calc(100% - 40px);
+  margin: 0 auto;
+  position: relative;
+
+  .switcher{
+    min-width: 312px;
+    position: absolute;
+    top: 55px;
+    right: 0;
+
+    @include only-phone{
+      right: 0;
+      left: 0;
+      margin: 0 auto;
+    }
+  }
+
+  .switcher-form{
+    min-width: 312px;
+    position: absolute;
+    top: 55px;
+    right: 0;
+
+    @include only-phone{
+      right: 0;
+      left: 0;
+      margin: 0 auto;
+    }
+  }
+}
```

Comment:
I'm against of increasing of the amount of global styles. Please implement separate components instead.

Reply:
@artem-rozhkov-dev-optima consult @sadiqevani  on this